src/components/DropdownSelect/index.js

```diff
@@ -85,32 +85,51 @@ const Icon = ({ network }) => {
 const DropdownSelect = ({
   options,
   active,
+  disabled,
   setActive,
   color,
   width = null,
 }) => {
   const [showDropdown, toggleDropdown] = useState(false);
   const dropdownRef = useRef(null);
   const containerRef = useRef(null);
-  useClickAway(dropdownRef, event => {
+  useClickAway(dropdownRef, (event) => {
     if (showDropdown && !containerRef.current.contains(event.target))
       toggleDropdown(false);
   });
 
   return (
-    <Wrapper open={showDropdown} color={color} ref={containerRef} width={width}>
-      <RowBetween
-        onClick={() => toggleDropdown(!showDropdown)}
-        justify="center"
-      >
-        <TYPE.main display="flex">
-          <Icon network={active} />
-          {active}
-        </TYPE.main>
-        <StyledIcon>
-          <ArrowStyled />
-        </StyledIcon>
-      </RowBetween>
+    <Wrapper
+      open={showDropdown}
+      color={color}
+      ref={containerRef}
+      width={width}
+      disabled={disabled}
+    >
+      {disabled ? (
+        <RowBetween justify="center">
+          <TYPE.main display="flex" color={"disabled"}>
+            <Icon network={active} disabled={true} />
+            {active}
+          </TYPE.main>
```

Comment:
```suggestion
          <TYPE.main color={"disabled"}>
            {active}
          </TYPE.main>
```

Reply:
Isn't it better to keep the icon even if the dropdown is disabled?

Reply:
Because the <Icon/> will give you anyway null for non network props